atomicDockerLVM.md

```diff
@@ -104,11 +103,11 @@ drwxr-xr-x. 3 root root       26 Feb 27 20:39 ..
 -rw-r--r--. 1 root root 51200000 Feb 27 20:40 data
 ```
 
-And the space in `df -h` on the host will have increased.
+The disk usage shown by `df -h` on the host will have increased, even after deleting the container.
```

Comment:
@mattreid while you are editing this lab, it may be beneficial to add sample output from `df -h` before and after the container is deleted.